content/en/blog/community/spinkube-kind-rd/index.md

```diff
@@ -0,0 +1,328 @@
+  default_runtime_name = "runc"
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+  # set default runtime handler to v2, which has a per-pod shim
+  runtime_type = "io.containerd.runc.v2"
+  # Generated by "ctr oci spec" and modified at base container to mount poduct_uuid
+  base_runtime_spec = "/etc/containerd/cri-base.json"
+  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
+    # use systemd cgroup by default
+    SystemdCgroup = true
+
+# Setup a runtime with the magic name ("test-handler") used for Kubernetes
+# runtime class tests ...
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.test-handler]
+  # same settings as runc
+  runtime_type = "io.containerd.runc.v2"
+  base_runtime_spec = "/etc/containerd/cri-base.json"
+  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.test-handler.options]
+    SystemdCgroup = true
+
+[plugins."io.containerd.grpc.v1.cri"]
+  # use fixed sandbox image
+  sandbox_image = "registry.k8s.io/pause:3.7"
+  # allow hugepages controller to be missing
+  # see https://github.com/containerd/cri/pull/1501
+  tolerate_missing_hugepages_controller = true
+  # restrict_oom_score_adj needs to be true when running inside UserNS (rootless)
+  restrict_oom_score_adj = false
+  
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.spin]
+  runtime_type = "/usr/local/bin/containerd-shim-spin-v2"
+
+```
+
+> NOTE: this file is a copy of the original one that can be found inside the KinD container. The only addition to the file is the declaration of the `spin` plugin (the last 2 lines)
+
+### KinD config file
+
+The second configuration file is related to KinD and will be used when creating a new cluster:
+
+- Create a file in your `$HOME` directory called `kind-spin.yaml` (for example)
+  - You can create it inside a directory, however it still should be located in your `$HOME` directory
+  - The location will be important when creating the KinD cluster
+
+**Windows Users ONLY**
+
+```yaml
+@"
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraMounts:
+  - hostPath: /root/kind-spin/config.toml
+    containerPath: /etc/containerd/config.toml
+  - hostPath: /usr/local/containerd-shims/containerd-shim-spin-v2
+    containerPath: /usr/local/bin/containerd-shim-spin-v2
+"@ | rdctl shell -- tee -a /root/test.yaml
+```
+
+**Linux and MacOS Users ONLY**
+
+```yaml
+cat <<EOF > $HOME/kind-spin.yaml
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraMounts:
+  - hostPath: ${HOME}/config.toml
+    containerPath: /etc/containerd/config.toml
+  - hostPath: /usr/local/containerd-shims/containerd-shim-spin-v2
+    containerPath: /usr/local/bin/containerd-shim-spin-v2
+EOF
+```
+
+#### Connecting the dots (2)
+
+Rancher Desktop leverages 2 different technologies depending on the OS its installed.
+
+On Windows, WSL2 will be used and for Linux and MacOS, Lima is the preferred choice.
```

Comment:
maybe it's worth adding a link to WSL2 and Lima?

Reply:
Addressed in [bc85128](https://github.com/spinkube/documentation/pull/143/commits/bc851287ff5d8cc87a364a71989b2e79515dcb79)